playwright-tests/util/rpcmock.js

```diff
@@ -342,6 +343,30 @@ export async function updateDaoPolicyMembers({
       originalResult = instanceAccount.includes("testing")
         ? getNewPolicy(votePolicy, votePolicy, votePolicy)
         : getOldPolicy(votePolicy, votePolicy, votePolicy);
+      if (hasAllRole) {
+        originalResult.roles.push({
+          name: "all",
+          kind: "Everyone",
+          permissions: [
+            "call:AddProposal",
```

Comment:
Just a question, so that I understand this better. What is the reason for assigning all of these permissions here to Everyone? Are permissions for updating config/policy/parameteres normally given to everyone like it is here?

Reply:
It's not common, but definitely possible, user when creates the dao through other UI, allow everything for anyone in the beginning, I added it in tests just to make sure the UI handles everything as expected even it's uncommon